README.md

```diff
@@ -11,34 +13,34 @@
 This repository contains a number of tools from the [Open Force Field Initiative](http://openforcefield.org) for the development and use of modern molecular mechanics forcefields based on direct chemical perception and parameterized with rigorous statistical methods.
 
 This repository hosts tools that we have committed to stably maintain throughout their lifetimes:
-* Tools for using the [SMIRKS Native Open Force Field (SMIRNOFF) specification](https://open-forcefield-toolkit.readthedocs.io/en/topology/smirnoff.html)
+* Tools for using the [SMIRKS Native Open Force Field (SMIRNOFF) specification](https://open-forcefield-toolkit.readthedocs.io/en/latest/smirnoff.html)
 * Tools for direct chemical environment perception and manipulation
 
 ## Documentation
 
-[Documentation](https://open-forcefield-toolkit.readthedocs.io/en/topology) for the `openforcefield` toolkit is hosted at [readthedocs](https://open-forcefield-toolkit.readthedocs.io/en/topology).
+[Documentation](https://open-forcefield-toolkit.readthedocs.io/en/latest/) for the `openforcefield` toolkit is hosted at [readthedocs](https://open-forcefield-toolkit.readthedocs.io/en/latest).
 
 ## Installation
 
 `openforcefield` is a Python toolkit, and supports Python 2.7, 3.5, and 3.6.
```

Comment:
Is this list of supported Pythons accurate? I believe `conda-forge` only supports 2.7, 3.6, and 3.7 now, so we cannot support 3.5. Additionally, do we still support 2.7?

Reply:
Nice catch -- Currently on Travis we only test 3.6 and 3.7. Updated in 15b037b